### ISSUE TITLE
Menubar icon instead of text

### ISSUE THREAD

**Issue by michaelchadwick** [paulyoung/Statusfy#24]
Most menubar applications have a small icon to represent their status (for space reasons), rather than a string of text, so it'd be awesome if this app did, too :) The "title - artist" string could appear on click instead.

**Comment by benetis** [comment]
+1

**Comment by z3t0** [comment]
Just a thought, wouldn't this sort of be counter intuitive as you would no longer be able to see the song at a glance? Although I am using Bartender to clear up my menu bar in any case so that might explain my workflow.

**Comment by kihaki** [comment]
+1

Best case scenario would be:
- Able to switch between text view and icon via menu item
- Show a playing/paused icon instead of (playing)/(paused), afaik those icons exist as unicode characters, so they shouldn't be hard to implement

**Comment by z3t0** [comment]
Or even using font awesome would be nice for consistency

**Comment by wpcarro** [comment]
The option to toggle between an icon display and a textual displays may also help ameliorate [this issue](https://github.com/paulyoung/Statusfy/issues/16).

**Comment by z3t0** [comment]
Yes, maybe also making it auto-switch when there is not enough space?

**Comment by z3t0** [comment]
I doubt this project is in active development? If necessary I can go give it a shot, assuming there is enough interest

**Comment by paulyoung** [comment]
Thanks for the suggestion. I _did_ experiment with unicode when I added the player status but something about it didn't look quite right.

I'm open to seeing how it looks again, but I'd rather not support both text and unicode.

Also, I'm not interested in adding more options to hide the text behind an icon until clicked and things of that nature, but thanks again.
